django_mailbox/google_utils.py

```diff
@@ -17,37 +20,45 @@ class RefreshTokenNotFound(Exception):
 
 
 def get_google_consumer_key():
+    if SocialApp:
```

Comment:
I believe this should be a configuration option as using django-allauth and django-guardian in one application doesn't necessarily mean that django-allauth provides a secret for django-guardian.